From 7beaa57cacca2b2d871a2fdb83b6244679322f2d Mon Sep 17 00:00:00 2001
From: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@intel.com>
Date: Wed, 16 Jul 2014 16:46:44 -0700
Subject: i2c: dw_i2c: Remove extra warning message

Using both pr_error() and WARN_ON() calls on failure case
is unnecessary. So remvoing the warning call.

Signed-off-by: Kuppuswamy Sathyanarayanan <sathyanarayanan.kuppuswamy@intel.com>
---
 arch/x86/platform/intel-mid/intel_mid_pcihelpers.c |    1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/x86/platform/intel-mid/intel_mid_pcihelpers.c b/arch/x86/platform/intel-mid/intel_mid_pcihelpers.c
index f4aafb0..004a8b2 100644
--- a/arch/x86/platform/intel-mid/intel_mid_pcihelpers.c
+++ b/arch/x86/platform/intel-mid/intel_mid_pcihelpers.c
@@ -248,7 +248,6 @@ int intel_mid_dw_i2c_acquire_ownership(void)
 			pr_err("PUNIT SEM: %d\n",
 					intel_mid_msgbus_read32(PUNIT_PORT,
 						PUNIT_SEMAPHORE));
-			WARN_ON(1);
 			return ret;
 		}
 	}
-- 
1.7.9.5

